### PR TITLE
Arm adjust

### PR DESCRIPTION
**Pull request by CaleWineinger**

### CODE REVIEW

File: 2024 Code/src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -47,8 +48,8 @@ private void configureBindings() {
     SubsystemContainer.swerveSubsystem.setDefaultCommand(m_swerveDefaultCommand);
     SubsystemContainer.intakeSubsystem.setDefaultCommand(m_intakeDefaultCommand);
     SubsystemContainer.climbSubsystem.setDefaultCommand(m_climbDefaultCommand);
-    SubsystemContainer.armSubsystem.setDefaultCommand(new ArmAngleCommand(50));
-    SubsystemContainer.ShooterSubsystem.setDefaultCommand(new ShooterDefaultCommand());
+    SubsystemContainer.armSubsystem.setDefaultCommand(new ArmAngleCommand(0));
+    SubsystemContainer.ShooterSubsystem.setDefaultCommand(m_shooterDefaultCommand);
```
Comment (gurellia53) [inline 2024 Code/src/main/java/frc/robot/RobotContainer.java:52]: nah, just do `new ShooterDefaultCommand() ` here

Comment (CaleWineinger) [reply 2024 Code/src/main/java/frc/robot/RobotContainer.java:52]: Thats why it didn't build

File: 2024 Code/src/main/java/frc/robot/commands/ArmManualCommand.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.Constants.ArmConstants;
+import frc.robot.util.SubsystemContainer;
+
+public class ArmManualCommand extends Command {
+  private double m_shiftAngle;
+  private XboxController m_controller;
+  private double m_targetAngle;
```
Comment (gurellia53) [inline 2024 Code/src/main/java/frc/robot/commands/ArmManualCommand.java:16]: Unless I'm missing something, I don't think m_targetAngle needs to be a member variable. just make it a double in the execute method.

Comment (CaleWineinger) [reply 2024 Code/src/main/java/frc/robot/commands/ArmManualCommand.java:16]: Wouldn't that recreate it everytime?

File: 2024 Code/src/main/java/frc/robot/commands/ArmManualCommand.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  }
+
+  @Override
+  public void execute() {
+    m_targetAngle = 0;
+    if (Math.abs(m_controller.getLeftX()) > ArmConstants.ARM_ADJUST_DEADZONE) {
+      m_shiftAngle += m_controller.getLeftX() / 10;
+    }
+    m_targetAngle = SubsystemContainer.armSubsystem.armPID(m_shiftAngle);
+    SubsystemContainer.armSubsystem.armChecker(m_targetAngle);
```
Comment (gurellia53) [inline 2024 Code/src/main/java/frc/robot/commands/ArmManualCommand.java:35]: not for this review, but I'm thinking armChecker should be moved into the subsystem so every command doesn't need to call it.

Comment (CaleWineinger) [reply 2024 Code/src/main/java/frc/robot/commands/ArmManualCommand.java:35]: Yeah I was thinking about that on my way home

File: 2024 Code/src/main/java/frc/robot/commands/ArmManualCommand.java (hunk 2 above)
Comment (gurellia53) [reply 2024 Code/src/main/java/frc/robot/commands/ArmManualCommand.java:16]: yeah, but you're setting it to 0 every time anyways.

**Review by gurellia53 — APPROVED** [review]

**Review by Noah9847 — APPROVED** [review]

**Pull request merged**